src/plugins/liveobjects/statemessage.ts

```
@@ -144,41 +150,24 @@ export class StateMessage {
-      entry.data.encoding = encoding ?? undefined;
-
-      if (error) {
-        throw error;
-      }
-    };
-
     if (message.object?.map?.entries) {
-      for (const entry of Object.values(message.object.map.entries)) {
-        await decodeMapEntry(entry, inputContext, decodeDataFn);
-      }
+      await this._decodeMapEntries(message.object.map.entries, inputContext, decodeDataFn);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)